bookx/settings.py

```diff
@@ -136,11 +131,17 @@
 MEDIA_ROOT = os.path.join(BASE_DIR, "media")
 
 AUTH_USER_MODEL = "users.CustomUser"
-AUTHENTICATION_BACKENDS = [
-    "users.backends.EmailBackend",
-    "django.contrib.auth.backends.ModelBackend",
-]
+# AUTHENTICATION_BACKENDS = [
+#     "users.backends.EmailBackend",
+#     "django.contrib.auth.backends.ModelBackend",
+# ]
+AUTHENTICATION_BACKENDS = ["django.contrib.auth.backends.ModelBackend"]
+
 EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+# EMAIL_BACKEND = "sendgrid_backend.SendgridBackend"
```

Comment:
It's best not to leave commented out code, it kind of beats the purpose of git ;)

Also, if SENDGRID config is only needed on production, maybe worth removing it from here too?